FirebaseMessaging/Sources/Token/FIRMessagingTokenManager.m

```diff
@@ -300,6 +307,14 @@ - (void)fetchNewTokenWithAuthorizedEntity:(NSString *)authorizedEntity
           handler(nil, error);
           return;
         }
+
+        if (!self) {
+          NSError *noSelf = [NSError messagingErrorWithCode:kFIRMessagingErrorCodeUnknown
```

Comment:
Same here, just for style naming convention

Reply:
Fixed this - renamed to lostRefError.